nixos/modules/system/activation/switchable-system.nix

```diff
@@ -5,101 +5,49 @@
   ...
 }:
 
-let
-  perlWrapped = pkgs.perl.withPackages (
-    p: with p; [
-      ConfigIniFiles
-      FileSlurp
-    ]
-  );
-in
 {
-  options.system.switch = {
-    enable = lib.mkOption {
-      type = lib.types.bool;
-      default = true;
-      description = ''
-        Whether to include the capability to switch configurations.
+  imports = [
+    (lib.mkRemovedOptionModule [ "system" "switch" "enableNg" ] ''
```

Comment:
Removing this option unnecessarily crashes eval on peoples machines. It could have been kept around for at least a while with a warning that it is going to be removed.